episodes/recipe_instructions.Rmd

```diff
@@ -24,7 +24,7 @@ exercises: 2
 
 - Use a box grater to shred the cold butter into a large mixing bowl.
 - Add the flour to the bowl and use a fork, bench scraper, or pastry cutter (if you have one) to cut the butter into the flour.
-- Add the sourdough starter discard, buttermilk, honey and salt.
+- Add the sourdough starter discard (the stuff that you would normally throw away after feeding your sourdough starter), buttermilk, honey and salt.
```

Comment:
I think we should link to a external resource here, where the learner can find out more about sourdough starters.